doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
 
-The boolean `worker.suicide` lets you distinguish between voluntary and accidental
-exit, the master may choose not to respawn a worker based on this value.
+The boolean `worker.exitedVoluntarily` lets you distinguish between voluntary
+and accidental exit, the master may choose not to respawn a worker based on
```

Comment:
Nit: Maybe fix the comma-splice while we're in here? (Or not if we're at bikeshed burnout point.) Anyway, period/full-stop after `exit` and have `the master may choose` be the start of the next sentence.